tests/Features/Commands/GitleaksCommandsTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EcEuropa\Toolkit\Tests\Features\Commands;
+
+use EcEuropa\Toolkit\TaskRunner\Commands\DocumentationCommands;
+use EcEuropa\Toolkit\TaskRunner\Commands\GitleaksCommands;
+use EcEuropa\Toolkit\Tests\AbstractTest;
+
+/**
+ * Test Gitleaks commands.
+ *
+ * @group toolkit
+ */
+class GitleaksCommandsTest extends AbstractTest
+{
+
+    /**
+     * Data provider for testToolkit.
+     *
+     * @return array
+     *   An array of test data arrays with assertions.
+     */
+    public function dataProvider()
+    {
+        return $this->getFixtureContent('commands/gitleaks.yml');
+    }
+
+    /**
+     * Test ToolkitCommands commands.
+     *
+     * @param string $command
+     *   A command.
+     * @param array $expectations
+     *   Tests expected.
+     *
+     * @dataProvider dataProvider
+     */
+    public function testGitleaks(string $command, array $expectations = [])
+    {
+        // Run command.
+        $result = $this->runCommand($command);
+//        $this->debugExpectations($result['output'], $expectations);
```

Comment:
Please clean up the code here.

Reply:
Hi @joaocsilva ,
That line is for debug, at the moment is present in all *Test.php classes.
To be removed in a separate ticket.